src/js/components/Anchor.js

```diff
@@ -10,7 +11,12 @@ const Anchor = props => {
   let icon;
 
   if (props.primary) {
```

Comment:
Let's add the icon even in the non-primary case as well. That will allow using icons in non-primary contexts.